README.md

```diff
@@ -1,76 +1,39 @@
-![CF](https://camo.githubusercontent.com/70edab54bba80edb7493cad3135e9606781cbb6b/687474703a2f2f692e696d6775722e636f6d2f377635415363382e706e67) 13: Single Resource Mongo and Express API
-===
+Lab 13: Single Resource Mongo and Express API
```

Comment:
No points deducted for this README, but remember the instructions also said to detail the response for valid AND invalid requests.